server.js

```diff
@@ -108,11 +122,15 @@ app.post('/', function(req, res) {
     }, function(err, configRes) {
       // default config
       var repoConfig = {
-        userBlacklist: []
+        userBlacklist: [],
+        maxReviewers: 3
       };
 
       if (!err && configRes) {
-        try { repoConfig = JSON.parse(configRes); } catch (e) {}
+        try {
+          merge(repoConfig, JSON.parse(configRes));
```

Comment:
What do I need to do to enable this syntax? Everything I've read seems to indicate it should be enabled by default, but that doesn't seem to be the case.